modules/core/src/main/java/org/apache/ignite/internal/processors/management/ManagementConsoleProcessor.java

```diff
@@ -35,4 +35,9 @@
      * @return Management configuration instance.
      */
     public ManagementConfiguration configuration();
+
+    /**
+     * @return Control Center sender.
+     */
+    public ControlCenterSender sender();
```

Comment:
Please clarify, that this sender is responsible for communication with the backend.